extensions/api/data-management/transferprocess/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/service/TransferProcessService.java

```diff
@@ -76,4 +79,13 @@
      */
     @NotNull
     ServiceResult<TransferProcess> deprovision(String transferProcessId);
+
+    /**
+     * Initiate transfer request.
+     *
+     * @param request for the transfer.
+     * @return id of created transferProcess.
+     */
+    @NotNull
+    String initiateTransfer(DataRequest request);
```

Comment:
I know that at the moment there's no "failed case", but as the other "command" methods on the other service interfaces, this should return a `ServiceResult<String>` to being able to handle an eventual failure

Reply:
I was thinking about it but then as there's no "failed case" I went for simpler option to return plain String, but it makes sense to make the interface methods consistent. Changed as you suggested